c2corg_ui/static/js/map/diffmap.js

```diff
@@ -81,9 +80,10 @@ app.DiffMapController = function(mapFeatureCollection) {
   app.utils.setupSmartScroll(mouseWheelZoomInteraction);
 
   if (!mapFeatureCollection) {
-    this.map.setView(new ol.View({
-      center: app.MapController.DEFAULT_CENTER,
-      zoom: app.MapController.DEFAULT_ZOOM
+    var mapSize = this.map.getSize() || null;
+    this.map.getView().fit(app.MapController.DEFAULT_EXTENT, mapSize,
```

Comment:
Maybe not, indeed.
Actually it's already like this in the code you have written: https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/static/js/map/diffmap.js#L168-L172

Perhaps we don't really need to have ``|| null`` at line 83, but I guess it's required by the Closure Compiler so that mapSize is ``ol.Size|null``?